app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php

```diff
@@ -381,37 +381,105 @@ public function validateEstimatedArea(Request $request)
         );
     }
 
-    public function getPolygonsAsGeoJSON()
+    public function getPolygonAsGeoJSONDownload(Request $request)
     {
-        $limit = 2;
-        $polygons = PolygonGeometry::selectRaw('ST_AsGeoJSON(geom) AS geojson')
-          ->orderBy('created_at', 'desc')
-          ->whereNotNull('geom')
-          ->limit($limit)
-          ->get();
-        $features = [];
-
-        foreach ($polygons as $polygon) {
-            $coordinates = json_decode($polygon->geojson)->coordinates;
+        try {
+            $uuid = $request->query('uuid');
+
+            $polygonGeometry = PolygonGeometry::where('uuid', $uuid)
+            ->select(DB::raw('ST_AsGeoJSON(geom) AS geojsonGeom'))
+            ->first();
```

Comment:
These two lines should be indented. I'm surprised the linter didn't change it 🤷🏻

Reply:
done